opsgenie_alert_tagging.rb

```diff
@@ -0,0 +1,4 @@
+require './lib/opsgenie_alert_tagging'
+require './lib/opsgenie_alert'
+
+OpsgenieAlertTagging.new(ARGV[0]).call
```

Comment:
What happens when you import this file by mistake? Would it be worth guarding to only actually execute anything if this is the file we're running?